imports/plugins/included/connectors-shopify/client/settings/shopify.js

```diff
@@ -39,6 +39,97 @@ Template.shopifyProductImport.events({
   }
 });
 
+Template.shopifySync.helpers({
+  activeWebhooks() {
+    const pkg = Packages.findOne({
+      name: "reaction-connectors-shopify",
+      shopId: Reaction.getShopId()
+    });
+
+    if (pkg && pkg.settings && Array.isArray(pkg.settings.webhooks) && pkg.settings.webhooks.length > 0) {
+      return true;
+    }
+    return false;
+  },
+
+  integrationIsActive(integration) {
+    const pkg = Packages.findOne({
```

Comment:
Any reason this isn't updated to the pattern:

```
    const { settings } = Reaction.getPackageSettings( reaction-connectors-shopify");
```

Reply:
Nope, will update.